database/consts.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Dict
+
+OBJECT_TYPE = Any
+QUERY_TYPE = Dict[str, OBJECT_TYPE]
```

Comment:
Type aliases, especially when used with typing module, should be written in CamelCase, starting with a capital letter. 

`Dict[str, Any]`